Loop Widget Extension/Widgets/SystemStatusWidget.swift

```diff
@@ -26,7 +26,7 @@ struct SystemStatusWidgetEntryView: View {
         } else {
             let mostRecentGlucoseDataDate = entry.mostRecentGlucoseDataDate ?? Date().addingTimeInterval(.minutes(16))
             let mostRecentPumpDataDate = entry.mostRecentPumpDataDate ?? Date().addingTimeInterval(.minutes(16))
-            age = abs(max(min(0, mostRecentGlucoseDataDate.timeIntervalSinceNow), min(0, mostRecentPumpDataDate.timeIntervalSinceNow)))
+            age = max(abs(min(0, mostRecentPumpDataDate.timeIntervalSinceNow)), abs(min(0, mostRecentGlucoseDataDate.timeIntervalSinceNow)))
```

Comment:
Nit: If you want to think in terms of "age", which is generally positive for these kinds of things, then I often switch it around and do `now.timeIntervalSince(mostRecentPumpDataDate)`, which should generally be positive. 

Maybe something like this:
```
let now = Date()
let glucoseDataAge = entry.mostRecentGlucoseDataDate?.timeIntervalSince(now) ?? 0
let pumpDataAge = entry. mostRecentPumpDataDate?.timeIntervalSince(now) ?? 0
let age = min(glucoseDataAge, pumpDataAge)
```

I think if for some bizarre reason mostRecentGlucoseDataDate or mostRecentPumpDataDate were in the future, a negative age is fine for LoopCompletionFreshness.

Reply:
I mirrored what we were using the in the if block above for the lastLoopCompleted. Happy to switch to using this method if that's what you'd prefer!

Reply:
This is a small thing; whatever your preference is fine.

Reply:
Okay -- I'll leave it as-is in that case. I've done a fair amount of testing with this and since this is fixing some unwanted behavior of already merged code I'd like to get this out asap. Thanks for the feedback!